### DISCUSSION TITLE
KeyBinding priority execution and Event

### DISCUSSION THREAD

**Discussion by Tenjim** [AvaloniaUI/Avalonia#9027] [Q&A] [answered]
Hey guys,

I have a question about Event and Keybinding. 
I have a Keybinding on my main view and I have an Keydown Event to manage other key functionnalities in child Usercontrol,.

What I can see is that KeyBinding execute first than my KeyDown Event. Can we change priority's Keybinding ?

Thanks in Advance.

**Comment by maxkatz6** [comment, marked as answer]
Hi!
No, from what I see it's not possible.
KeyBindings are handled before raising KeyDown event on any control.

**Reply by Tenjim** [reply]
This behavior can be an Enhancement ?

**Comment by Tenjim** [comment]
Thank for answer. 

On my side, I manage with that workaround:

Create an EventHandler on root UserControl and manage all keybinding on that.
